connexion/security/aiohttp_security_handlers_factory.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+import functools
+import logging
+
+import aiohttp
+
+from ..exceptions import OAuthProblem, OAuthResponseProblem, OAuthScopeProblem
+from .security_handler_factory import SecurityHandlerFactory
+
+logger = logging.getLogger('connexion.api.security')
+
+
+class AioHttpSecurityHandlerFactory(SecurityHandlerFactory):
```

Comment:
are we doing to have to write one of these for every framework we support?
Is there a more generic version of this like `AsyncSecurityHandlerFactory`?

Reply:
done